### DISCUSSION TITLE
Implement getCountFromServer

### DISCUSSION THREAD

**Discussion by hyungtaecf** [firebase/flutterfire#10236] [Feature request] [answered]
It always was needed to read each document to get the count of documents in a collection. That is very expensive.
But I found that a few months ago, Firestore had introduced a feature "getCountFromServer()" that charges one read to get the count of each 1000 documents. And it is implemented for web, node, swift, and Kotlin but I could not find any implementation for Flutter. It is a very useful feature, so could it be implemented, please?

**Comment by stact** [comment, marked as answer]
Hey @hyungtaecf 

Seems to be already implemented

```dart
final snapshot = await yourCollection.count().get();
log(snapshot.count) 
```

**Reply by hyungtaecf** [reply]
Thank you so much! I just realized I was checking it at outdated dependencies...
